scipy-notebook/Dockerfile

```diff
@@ -8,7 +8,7 @@ USER root
 
 # libav-tools for matplotlib anim
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends libav-tools && \
+    apt-get install -y --no-install-recommends libav-tools python-pip python-dev build-essential && \
```

Comment:
Is there a reason to add OS packages for pip and python when the new installs are coming from conda?